content/blog/2018/08/2019-08-31-speaker-blog-delivery-automation.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Cultural Transformation via Delivery Automation"
+tags:
+- jenkins
+- jenkinsworld
+- jenkinsworld2018
+author: tanskann
+---
+
+NOTE: This is a guest blog by
+link:https://devopsworldjenkinsworld2018.sched.com/speaker/christian_weber.1y99b7id[Christian Weber] of
+link:https://github.com[GitHub] and
+link:https://devopsworldjenkinsworld2018.sched.com/speaker/tracy_ragan.70xae17[Tracy Ragan] of
+link:https://www.deployhub.com/[DeployHub].
+
+
+
+image::/images/conferences/devops-world-2018.jpg[DevOps World | Jenkins World 2018, float="left", link="https://www.cloudbees.com/devops-world"]
+
+Your Jenkins CI orchestration is all about workflow processing for your Dev process, but now you want to push continuous delivery.
+This of course means you need to have the pre-production and production teams on board with automation.
+Seems like an easy task, but you will soon find that cultural transformation in organizations is hard.
+Really hard.
+Whether you work in a large, small or somewhere in between organization, teams continuously struggle with changing cultural mindset to one of automation, inclusion and efficient workflow and design.
+
+
+So where do you start?
+Choosing the right tools can streamline this process, but how can you get people on board?
+And what are the risks if they choose not to follow your path to a fully automated process across the pipeline?
+
+Communicating the full benefits of what you have achieved by automating your development processes is a good place to start.
+But, by now most organizations understand the benefit of agile practices and its ability to deliver new innovation to end users at a rapid pace.
+And therein lies the real issue.
+Most organizations have not seen the promise of agile due mainly in part to downstream practices that continue to be rooted in waterfall methods.
+While agile practices have brought testing into the same room as developers, this is not always the case for production control.
+Not everyone has adopted the concept of a Site Reliability Engineer, someone who manages production, but works on the same Agile team as the developers.
+Most production deployments are still managed by throwing updates over the proverbial wall and waiting for the production team to execute their own release process to the production environment.
+And it may not be fast or agile.

```

Comment:
Many of these sentences starting with "And" or "But" look weird in a written form. Would be surprised if this is condoned by common English style guides.